numbat/src/unit.rs

```diff
@@ -209,12 +230,15 @@ impl Unit {
         self == &Self::scalar()
     }
 
-    pub fn new_base(name: &str, canonical_name: &str) -> Self {
+    pub fn new_base(name: &str, canonical_name: &str, accepts_prefix: AcceptsPrefix) -> Self {
```

Comment:
This could directly accept a `CanonicalName`, but it'd make the tests a tad more verbose so it might be fine as-is.

Reply:
Thanks, it was a bit verbose, but not a big deal. Changed.